src/main/java/com/jcabi/dynamodb/maven/plugin/Instances.java

```diff
@@ -141,21 +143,20 @@ private void shutdown() {
      */
     private static Process process(final File dist, final int port,
         final File home, final List<String> args) throws IOException {
-        return new ProcessBuilder().command(
-            new String[] {
-                new File(home, "bin/java").getAbsolutePath(),
-                new StringBuilder("-Djava.library.path=")
-                    .append(dist)
-                    .append(System.getProperty("path.separator"))
-                    .append(new File(dist, "DynamoDBLocal_lib"))
-                    .toString(),
-                "-jar",
-                "DynamoDBLocal.jar",
-                "--port",
-                Integer.toString(port),
-                StringUtils.join(args, " "),
-            }
-        ).directory(dist).redirectErrorStream(true).start();
+        final int baseCommandCount = 6;
```

Comment:
@thecarlhall we don't need to worry about this count. just create a new arraylist without any arguments.